en/concept/flight_modes.md

```diff
@@ -1,34 +1,36 @@
 # Flight Modes
 
-**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md).
+**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md), and are loosely grouped according to their level of control of the vehicle by the system vs. the user: "manual" modes, where the controller is essentially a pass-through from the user's remote control to the vehicle's actuators; "assisted" modes, where the system's outputs are coordinated with the user's commands; and, "auto" modes, where the controller requires little to no user input whatsoever.
```

Comment:
I like the description. The definition of "manual" doesn't seem quite right - only MANUAL is strictly "pass-through". The others change the interpretation of the inputs somewhat to add a degree of compensation for the clumsiness of new fliers.